### ISSUE TITLE
Deep Network Comparison.ipynb cannot achive the accurity

### ISSUE THREAD

**Issue by xuyifeng-nwpu** [awjuliani/TF-Tutorials#7]
In the readme of " Deep Network Comparison.ipynb"  
With little parameter tuning I was able to get them to perform above 90% accuracy on a test set after only an hour or so.

The train phase had been pass 12 hours ,however the test set accuracy is so low.
-----------------------
Step: 400 Loss: 1.75564 Accuracy: 0.234375
Test set accuracy: 0.252
----------------------------

My environment:  
hardware: i7 cpu,  980ti nvidia card ,   24G 
software:  cuda7.5  ,tensorflow 1.1,   python 3,   cudnn5.0
